app/views/projects/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1><%= @project.name %></h1>
+
+<p>Material: <%= @project.material %></p><br>
+
+<p>Challenge Theme: <%= @challenge.theme %></p><br>
+
+<p>Number of Contestants: <%= @project.count_contestants %></p><br>
```

Comment:
From your reflection, you were talking about this usage - this is perfect and exactly what I would do too. Keep this up 👍